src/pages/instant-observability.js

```diff
@@ -814,9 +815,13 @@ const Label = ({ children, htmlFor }) => (
     htmlFor={htmlFor}
     css={css`
       display: block;
-      font-size: 12px;
-      font-weight: bold;
-      margin-bottom: 0.25rem;
+      font-size: 14px;
+      font-weight: 600;
+      margin-bottom: 1.5rem;
+
+      .light-mode & {
```

Comment:
I don't think we denote light mode as a class, it's just the default. We should reverse the logic here and default to light mode colors.

Reply:
That was a weird testing thing i was doing (not sure why, now that i think of it 🤣). Consider it gone 👍🏻